host/CMakeLists.txt

```diff
@@ -284,34 +284,34 @@ endif()
 # TODO: Replace these with `find_package` and add as dependencies to
 # the CMake package.
 if (UNIX)
-  if (NOT TARGET crypto)
+  if (NOT TARGET 3rdparty::crypto)
```

Comment:
I would prefer using the `openenclave` namespace for all oe libraries for a couple reasons.

1. 3rdparty is fairly generic and it is possible for a developer to include a project that also populates a 3rdparty namespace
2. openenclave is used for all other cmake namespaces in the project.